kusto/test/etoe/etoe_test.go

```diff
@@ -355,11 +355,21 @@ func TestFileIngestion(t *testing.T) {
 		panic(err)
 	}
 
+	secondaryClient, err := kusto.New(testConfig.SecondaryEndpoint, testConfig.Authorizer)
```

Comment:
This will make it hard to run tests locally.
Can we make these specific new tests optional?

Reply:
Look if that's what you had in mind